tests/test_update.py

```
@@ -0,0 +1,168 @@
+            patch("smolvm.cli.update._fetch_latest_from_pypi", return_value="1.0.0"),
+        ):
+            current, latest = _check_for_stable_update()
+            assert current == "0.9.0"
+            assert latest == "1.0.0"
+
+    def test_returns_none_on_network_failure(self) -> None:
+        with (
+            patch("smolvm.cli.update._get_current_version", return_value="1.0.0"),
+            patch("smolvm.cli.update._fetch_latest_from_pypi", return_value=None),
+        ):
+            current, latest = _check_for_stable_update()
```

Ruff (0.15.15)

[warning] Unpacked variable `current` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)